lib/diagnostic.js

```diff
@@ -38,19 +47,38 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Buddy',
+  species: 'Cat',
+  breed: 'shorthair',
+  vocalization: 'Meow',
+
+  vocalize: function() {
+    return 'Buddy says Meow';
+  },
+
+};
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function () {
-  /* Your answer here */
-};
+const Pet = function (name, species, breed, vocalization) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.vocalization = vocalization;
+
+  this.vocalize = vocalize;
+  };
```

Comment:
This brace closes ```function (name, ..., vocalization) {```, but the next one has no matching opening bracket and breaks the script.